### ISSUE TITLE
Enabling Prometheus endpoint makes tendermint crash

### ISSUE THREAD

**Issue by Fraccaman** [tendermint/tendermint#9581]
**Tendermint version** (use `tendermint version` or `git rev-parse --verify HEAD` if installed from source):
Tried with [0.37-alpha.2](https://github.com/tendermint/tendermint/releases/tag/v0.37.0-alpha.2) and [0.34.21](https://github.com/tendermint/tendermint/releases/tag/v0.34.21).
v0.37-alpha.2 crash but v0.34.21 works.

**Environment**:
- **OS** (e.g. from /etc/os-release): ubuntu/amazon linux/osx

**What happened**:
Tendermint is not able to produce blocks.

**What you expected to happen**:
It should start without crashing/receiving errors

**Have you tried the latest version**: yes

**How to reproduce it** (as minimally and precisely as possible):
```
tendermint init
modify `config.toml` in .tendermint/config to enable prometheus endpoint by setting `prometheus = true`
tendermint node --proxy_app=kvstore 
```

**Logs (paste a small part showing an error (< 10 lines) or link a pastebin, gist, etc. containing more of the log file)**:
https://pastebin.com/prwAfWbP

**Config (you can paste only the changes you've made)**:
`prometheus = true`

**Comment by zmanian** [comment]
I think this is because https://github.com/tendermint/tendermint/pull/9455 isn't included in 0.37-alpha

**Comment by Fraccaman** [comment]
I see, thanks!

**Comment by sergio-mena** [comment]
Fixed by #9455
(@Fraccaman feel free to reopen if you happen to see this problem showing up in a hash after #9455)
